Patas.Em.Harmonia.Domain/Models/Entities/Ngo.cs

```diff
@@ -1,11 +1,25 @@
 ﻿#nullable disable
 
-using Patas;
+using Patas.Em.Harmonia.Domain.Constants;
 
 namespace Patas.Em.Harmonia.Domain.Models.Entities
 {
     public partial class Ngo
     {
+        public Ngo() {}
```

Comment:
acho que o c# 11 já permite o uso do construtorvazio implicito, no caso sem necessidade de por, testa ai e me fala se não for pra eu não repercurtir essa maluquice

Reply:
Deixa não kkkkk eu tbm achei que dava, mas deu ruim ai tive que criar esse carinha, mas não sei se é só no caso do entity